Only in https-redirect: .terraform
diff -Ebur ../../terraform-google-lb-http/examples/https-redirect/main.tf https-redirect/main.tf
--- ../../terraform-google-lb-http/examples/https-redirect/main.tf	2020-12-09 21:47:43.000000000 +0530
+++ https-redirect/main.tf	2020-12-09 21:54:18.000000000 +0530
@@ -24,15 +24,27 @@
 
 resource "google_compute_network" "default" {
   name                    = var.network_name
-  auto_create_subnetworks = false
+  description             = "Default network for the project"
+  auto_create_subnetworks = true
+  timeouts {
+    create = null
+    delete = null
+    update = null
+  }
 }
 
 resource "google_compute_subnetwork" "default" {
   name                     = var.network_name
-  ip_cidr_range            = "10.127.0.0/20"
+  ip_cidr_range            = "10.160.0.0/20"
   network                  = google_compute_network.default.self_link
   region                   = var.region
-  private_ip_google_access = true
+  private_ip_google_access = false
+
+  timeouts {
+    create = null
+    delete = null
+    update = null
+  }
 }
 
 resource "google_compute_router" "default" {
@@ -73,6 +85,9 @@
     var.network_name,
     module.cloud-nat.router_name
   ]
+  
+  source_image_project  = "centos-cloud"
+  source_image_family   = "centos-8"
 }
 
 module "mig" {
@@ -91,7 +106,7 @@
 }
 
 module "gce-lb-http" {
-  source               = "../../"
+  source               = "github.com/kunal-g/terraform-google-lb-http?ref=issue-112-dual-stack"
   name                 = "ci-https-redirect"
   project              = var.project
   target_tags          = [var.network_name]
@@ -101,6 +116,15 @@
   use_ssl_certificates = true
   https_redirect       = true
 
+  ## ipv4 config
+  create_address       = true
+  ## ipv4 config
+
+  ## ipv6 config
+  enable_ipv6          = true
+  ipv6_address         = var.ipv6_address
+  ## ipv6 config
+
   backends = {
     default = {
       description                     = null
diff -Ebur ../../terraform-google-lb-http/examples/https-redirect/variables.tf https-redirect/variables.tf
--- ../../terraform-google-lb-http/examples/https-redirect/variables.tf	2020-12-09 21:47:43.000000000 +0530
+++ https-redirect/variables.tf	2020-12-09 22:02:56.000000000 +0530
@@ -29,3 +29,8 @@
 variable "project" {
   type = string
 }
+
+variable "ipv6_address" {
+  type    = string
+  default = "new"
+}
\ No newline at end of file
